pkg/initialize/run.go

```diff
@@ -60,20 +63,202 @@ func Run(isNonInitCmd bool) error {
+}
+
+func downloadCommercialPolicies(policyBasePath, accessToken string) error {
+	err := ensureDir(policyBasePath)
+	if err != nil {
+		return err
+	}
+
+	policyRepoPath := config.GetPolicyRepoPath()
+	err = os.MkdirAll(policyRepoPath, filePermissionBits)
+	if err != nil {
+		return err
+	}
+
+	const apiPath = "/v1/api/app/rules?default=true"
+	environment := config.GetPolicyEnvironment()
+
+	zap.S().Debugf("policy environment : %s", environment)
+	zap.S().Debugf("downloading commercial policies in %s", policyBasePath)
+
+	var client http.Client
+
+	req, err := http.NewRequest(http.MethodGet, environment+apiPath, nil)
+	if err != nil {
+		return fmt.Errorf("error constructing request object. error: '%v'", err)
+	}
+	req.Header.Add("Authorization", "Bearer "+accessToken)
+
+	res, err := client.Do(req)
+	if err != nil || res.StatusCode != 200 {
+		return fmt.Errorf("error downloading commercial policies. error: '%v', response status code: '%d'", err, res.StatusCode)
+	}
+	if res == nil {
+		return fmt.Errorf("error could not download policies, please check your network connection")
+	}
+
+	policies, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return fmt.Errorf("error reading api call response for commercial policies. error: '%v'", err)
+	}
+
+	return convertCommercialPolicies(policies, policyRepoPath)
+}
+
+func convertCommercialPolicies(policies []byte, policyRepoPath string) error {
+	var ruleMetadataList []commercialPolicyMetadata
+
+	err := json.Unmarshal(policies, &ruleMetadataList)
+	if err != nil {
+		return fmt.Errorf("failed to unmarshal policies into structure. error: '%v'", err)
+	}
+
+	for _, ruleMetadata := range ruleMetadataList {
+		policy, err := getCommercialPolicy(ruleMetadata)
+		if err != nil {
+			return err
+		}
+
+		err = saveCommercialPolicies(policy, policyRepoPath)
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func getCommercialPolicy(ruleMetadata commercialPolicyMetadata) (commercialPolicy, error) {
+	var policy commercialPolicy
+	var templateArgs map[string]interface{}
+
+	policy.regoTemplate = "package accurics\n\n" + ruleMetadata.RuleTemplate
+	policy.metadataFileName = ruleMetadata.RuleReferenceID + ".json"
+	policy.resourceType = ruleMetadata.ResourceType
 
+	policy.policyMetadata.Name = ruleMetadata.RuleName
+	policy.policyMetadata.File = ruleMetadata.RegoName + ".rego"
+	policy.policyMetadata.ResourceType = ruleMetadata.ResourceType
+	policy.policyMetadata.Severity = ruleMetadata.Severity
+	policy.policyMetadata.Description = ruleMetadata.RuleDisplayName
+	policy.policyMetadata.ReferenceID = ruleMetadata.RuleReferenceID
+	policy.policyMetadata.ID = ruleMetadata.RuleReferenceID
+	policy.policyMetadata.Category = ruleMetadata.Category
+	policy.policyMetadata.Version = ruleMetadata.Version
+
+	templateString, ok := ruleMetadata.RuleArgument.(string)
+	if !ok && templateString != "" {
+		return policy, fmt.Errorf("incorrect rule argument type, must be a []byte")
```

Comment:
you are asserting that RuleArgument is a string and logging that it must be []byte, is that correct?